ChangeLog

```diff
@@ -7,6 +7,9 @@ Features
      is functioning correctly.
    * Added a script to print build environment info for diagnostic use in test
      scripts, which is also now called by all.sh.
+   * Added a configuration file config-no-entropy.h that enables a subset of
+     library features that do not require an implementation of an entropy
```

Comment:
Please remove 'implementation of' to make it 'do not require an entropy source'.